libs/styles/styles.css

```diff
@@ -137,7 +137,7 @@ img {
 
 /* Not blocks */
 div:not([class]) > div > div img[src^="./media_"] {
-  width: 100%;
+  max-width: 100%;
```

Comment:
I think it would be valuable to get a bit more context on some use-cases around this, because the two can have quite different effects. A quick CodePen as a simulation - https://codepen.io/ramuntea/pen/eYPRJRe

If the image is larger than its wrapper, then the two declarations have the same effect. If, however, the image is smaller than its wrapper (which could have a set width), then the two declarations have different outcomes.

Reply:
@overmyheadandbody this issue is particularly about images in fragment. When previewed in fragment they load in their actual size. However with width: 100% when these fragments were added on the page, these images were getting stretched and distorted (see the before url). I did tried to add the specific selector like below and add max-width

```
/* fragment image */
.fragment > * img[src^="./media_"] {
  max-width: 100%;
}
```

but then both the css (the one for fragment and the more generic one too) got applied and image got stretched. For example the yellow flower image width stretched to 1274px from 183px
<img width="1278" alt="Screen Shot 2023-04-27 at 2 33 45 PM" src="https://user-images.githubusercontent.com/69535463/234996332-aefe5541-3f62-4583-ad6c-e5389589b24c.png">